src/components/Monaco/ReactMonacoEditor.tsx

```diff
@@ -56,6 +56,17 @@ export function ReactMonacoEditor({
     return true
   }, [editor])
 
+  const handleEditorMount = (
+    editor: monaco.editor.IStandaloneCodeEditor,
+    monaco: Monaco
+  ) => {
+    setEditor(editor)
+
+    if (props.onMount) {
+      props.onMount(editor, monaco)
+    }
```

Comment:
Now we're setting the instance of `editor` here but still respecting what the parent component is doing.